phira/locales/en-US/multiplayer.ftl

```diff
@@ -2,85 +2,85 @@
 multiplayer = Multiplayer
 
 connect = Connect
-connect-must-login = You must login to enter multiplayer mode
-connect-success = Connected successfully
-connect-failed = Failed to connect
-connect-authenticate-failed = Authorization failed
+connect-must-login = You must login to access multiplayer functionality.
+connect-success = Connected successfully.
+connect-failed = Failed to connect.
+connect-authenticate-failed = Authorization failed.
 reconnect = Reconnecting…
 
 create-room = Create Room
-create-room-success = Room created
-create-room-failed = Failed to create room
-create-invalid-id = Room ID consists of no more than 20 characters, including letters, numbers, - (dash) and _ (underscore)
+create-room-success = Room created.
+create-room-failed = Failed to create room.
+create-invalid-id = A Room ID should be 20 characters max, and only contain A-Z, 0-9, -, and _.
```

Comment:
Source code 
[src](https://github.com/TeamFlos/phira-mp/blob/main/phira-mp-common%2Fsrc%2Fcommand.rs#L79-L93)
According to the source code, lowercase letters are actually included, and should be described as well to avoid the assumption that only uppercase letters are allowed.